autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -49,9 +49,13 @@ def _fit(
             # with nan values.
```

Comment:
Other issues in this file:
1. Access to member `object_dtype_mapping` before its definition in line 433
2. Unused variable enc_columns in line 257